packages/loader/container-loader/src/loader.ts

```diff
@@ -89,6 +92,14 @@ export class RelativeLoader extends EventEmitter implements ILoader {
             const container = await this.resolve(request);
             return container.request(request);
         }
+
+        if (this.loader === undefined) {
+            return {
+                status: 400,
```

Comment:
A 404 might be more appropriate here (request is syntactically fine, but we can't locate the resource without a loader).  It's also more likely that people will have handling for the 404 already.